src/NetAppFiles/NetAppFiles/Az.NetAppFiles.psd1

```diff
@@ -125,7 +125,7 @@ PrivateData = @{
 * Snapshot creation date now named just ''created'''
 
         # Prerelease string of this module
-        # Prerelease = ''
+        Prerelease = 'preview'
```

Comment:
version 0.* is already considered preview, no need to add this field.

Reply:
> version 0.* is already considered preview, no need to add this field.

Yes. However there was confusion on this point recently resulting in an email thread between NetApp and @yuemlu and @paulomarquesc among others. It was mentioned that this field should be added but also that the documentation stating that is now outdated. Given that there have also been review comments on previous PS updates rejecting the changes as breaking (despite the version number) I thought I would add in the text to make it extra clear